.eslintrc.js

```diff
@@ -100,11 +100,27 @@ module.exports = {
     react: {
       version: '16',
     },
-  },
-  overrides: [
-    {
-      files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
-      extends: ['plugin:testing-library/react'],
+    settings: {
+      react: {
+        version: '16',
+      },
+      'import/resolver': {
+        typescript: {}, // this loads <rootdir>/tsconfig.json to eslint
+      },
     },
```

Comment:
Looks like we already have react prop in settings at line 99. Can we remove the one on line 99 - 102?

Reply:
oh you are right. when I updated my branch with upstream master this happened. I'll remove it.